### ISSUE TITLE
Add release notes

### ISSUE THREAD

**Issue by ddenisyuk** [mvel/mvel#116]
Add release notes to project. This will allow to understand changes and requirements for each release

**Comment by lichtin** [comment]
I agree. @mariofusco would it be possible to have very short bullet points for a release?

**Comment by Inverness** [comment]
It's a bit absurd that I can't find a change log anywhere.

We're using this for a critical piece of software and I'd like to know what the impact will be if we upgrade.
